src/main/java/duke.task/Event.java

```diff
@@ -0,0 +1,24 @@
+package duke.task;
+
+public class Event extends Task {
+    private String eventTime;
+
+    public Event(String taskInfo, String eventTime) {
+        super(taskInfo);
+        this.eventTime = eventTime;
+    }
+
+    public Event(String taskInfo, boolean isDone, String eventTime) {
```

Comment:
What is the purpose of having isDone as an argument here?

Reply:
This will be called when saved events are recalled from storage so as to ensure that they are marked appropriately.